packages/core-components/src/components/buttons/LanguageSelectButtonMobile.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+
+import {
+  type DropdownButtonProps,
+  Option,
+} from '@/components/buttons/DropdownButton';
+
+type LanguageSelectButtonMobileProps = DropdownButtonProps & {
```

Comment:
we should reuse the DropdownButtonProps this way. 
Firstly, this will cause confusion for code review. 
More than that, there are possibilities that DropdownButotn introduce a prop that this LanguageSelectButtonMobile won't use